tools/picard.py

```diff
@@ -116,20 +119,17 @@ def per_read_group(self, inBam, outDir, picardOptions=None, JVMmemory=None):
                 os.mkdir(outDir)
         else:
             picardOptions = picardOptions or []
-            opts = ['INPUT=' + inBam, 'OUTPUT_DIR=' + outDir, 'OUTPUT_PER_RG=true']
+            opts = ['INPUT=' + inBam, 'OUTPUT_DIR=' + outDir, 'OUTPUT_PER_RG=true', 'RG_TAG=ID']
```

Comment:
@dpark01: I can use your input here. Is this OK? Upgrading Picard from 1.126 to 1.141 changes the usage of `OUTPUT_PER_RG`, and seems to require `RG_TAG=ID` to conform to prior usage. The change seems to have appeared in [`1.127`](https://github.com/broadinstitute/picard/commit/4fef32ef397e33114a3711469a8c4e66507fafb9)